tests/test_imports.py

```diff
@@ -0,0 +1,37 @@
+"""Tests to verify package can be imported and Python version compatibility."""
+
+import sys
+
+
+def test_python_version():
+    """Verify Python version is 3.9 or higher."""
+    assert sys.version_info >= (3, 9), f"Python 3.9+ required, got {sys.version_info}"
+
+
+def test_package_import():
+    """Verify the package can be imported."""
+    import yaml_diffs
+
+    assert yaml_diffs is not None
+
+
+def test_package_version():
+    """Verify package version is accessible."""
+    from yaml_diffs import __version__
+
+    assert __version__ is not None
+    assert isinstance(__version__, str)
+    assert len(__version__) > 0
+
+
+def test_dependencies_importable():
+    """Verify key dependencies can be imported."""
+    import fastapi
+    import pydantic
+    import yaml
+
+    # Verify they have expected attributes/functionality
+    assert hasattr(pydantic, "BaseModel")
```

Comment:
Excellent improvement! Using `hasattr()` to verify actual functionality (BaseModel, safe_load, FastAPI) is much better than just checking for None. This ensures the dependencies are not just importable but actually functional.